src/pages/dashboard.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Box,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  useColorModeValue,
+  Tooltip,
+  useToast,
+  useBreakpointValue,
+} from '@chakra-ui/react';
+import { ArrowBackIcon, ChevronRightIcon, CopyIcon } from '@chakra-ui/icons';
+import { useRouter } from 'next/router';
+import Sidebar from './components/Sidebar';
+
+const Dashboard: React.FC = function Dashboard() {
+  const { t } = useTranslation('dashboard');
+  const router = useRouter();
+  const [apiKey, setApiKey] = useState('');
+  const toast = useToast();
+  const ml = useBreakpointValue({ base: 10, lg: 270 });
+  const pageHeaderDisplay = useBreakpointValue({ base: 'block', lg: 'flex' });
+
+  const handleCopyApiKey = () => {
+    setApiKey('example-api-key');
+    navigator.clipboard.writeText(apiKey);
+    toast({
+      title: t('API key copied'),
+      description: t('Your API key has been copied to your clipboard.'),
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    });
+  };
+
+  const handleGoBack = () => {
+    router.back();
+  };
+
+  return (
+    <Box minH="100vh" bg={useColorModeValue('white', 'white')}>
+      <Sidebar />
+      <Box ml={ml} pt={5} display="flex">
+        <Text
+          fontSize="lg"
+          fontFamily="monospace"
+          color="blue.400"
+          fontWeight="bold"
+          marginRight={3}
+          cursor="pointer"
+          onClick={handleGoBack}
+        >
+          <ArrowBackIcon
+            __css={{
+              boxSize: '1rem',
+              marginRight: '5px',
+            }}
```

Comment:
take a look through all your new code and remove the use of `__css` since it's not the pattern used in this project, just use the Chakra component props directly